doc/deployment/nginx_and_url_path_config.rst

```diff
@@ -0,0 +1,137 @@
+- **FORCE_SCRIPT_NAME**: Uses the value from ``BASE_PATH``, setting ``/tickets`` as the base path for URL routing within Django.
+
+This means that all URLs within Django are prefixed with ``/tickets``, allowing users to access ``eventyay-tickets`` via URLs like ``domain.com/tickets/``.
+
+NGINX Configuration
+-------------------
+
+The following NGINX configuration serves the ``eventyay-tickets`` application along with other services such as ``talk`` and ``video``. The configuration is hosted on ``app-test.eventyay.com``.
+
+Key Sections of NGINX Configuration
+-----------------------------------
+
+1. **Server Block for HTTPS**:
+   This block listens on port ``443`` for HTTPS traffic.
+   SSL certificates are configured here using Let's Encrypt for secure communication.
+
+   .. code-block:: nginx
+
+       server {
+           server_name app-test.eventyay.com;
+           listen 443 ssl;
+
+           ssl_certificate /etc/../fullchain.pem;
+           ssl_certificate_key /etc/../privkey.pem;
+           include /etc/../options-ssl-nginx.conf;
+           ssl_dhparam /etc/../ssl-dhparams.pem;
+       }
+
+2. **URL Path Configurations**:
+
+   - **Root Location**:
+     - Proxies requests from ``https://app-test.eventyay.com/`` to the backend on ``localhost:8455/common/``.
+
+   - **/tickets**:
+     - This location is for the ``eventyay-tickets`` application.
+     - Requests to ``https://app-test.eventyay.com/tickets/`` are proxied to ``localhost:8455/``.
+
+     .. code-block:: nginx
+
+         location /tickets/ {
+             proxy_pass http://localhost:8455/;
+             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+             proxy_set_header X-Forwarded-Proto $scheme;
+             proxy_set_header Host $host;
+         }
+
+   - **/talk** and **/video**:
+     - These locations handle other components (``talk`` and ``video``) by proxying requests to their respective backend services on different ports.
+
+3. **Custom Redirects**:
+   - **/video/admin**:
+     - Redirects requests from ``/video/admin/`` to the ``control`` panel at ``https://app-test.eventyay.com:8443/control``.
+
+     .. code-block:: nginx
+
+         location /video/admin/ {
+             return 301 https://app-test.eventyay.com:8443/control;
+         }
+
+   - **/talk/static** and **/media**:
+     - Routes static files for the ``talk`` component, setting up paths for ``/talk/media/`` and ``/media/`` to point to Docker volumes where media files are stored.
+
+     .. code-block:: nginx
+
+         location /talk/media/ {
+             rewrite ^/talk/media/(.*)$ /media/$1 break;
+             root /var/lib/docker/volumes/pretalx_pretalx-data/_data;
+         }
+
+4. **HTTP to HTTPS Redirection**:
+   - Redirects all HTTP requests on port ``80`` to HTTPS, ensuring secure communication.
+
+   .. code-block:: nginx
+
+       server {
+           listen 80;
+           server_name app-test.eventyay.com;
```

Comment:
The documentation should be URL agnostic. If we deploy this somewhere else it does not make sense.